src/Compilers/CSharp/Portable/Lowering/Instrumentation/DebugInfoInjector_SequencePoints.cs

```diff
@@ -28,17 +28,22 @@ internal static BoundStatement AddSequencePoint(VariableDeclaratorSyntax declara
             return result;
         }
 
-        internal static BoundStatement AddSequencePoint(PropertyDeclarationSyntax declarationSyntax, BoundStatement rewrittenStatement)
+#nullable enable
+        internal static BoundStatement AddSequencePoint(
```

Comment:
>AddSequencePoint [](start = 39, length = 16)

Consider making this method private, keeping ```internal static BoundStatement AddSequencePoint(PropertyDeclarationSyntax declarationSyntax, BoundStatement rewrittenStatement)```, adding similar overload for data property and having both delegate to the private method.